cnd/src/http_api/route_factory.rs

```diff
@@ -45,6 +45,11 @@ pub fn create(
         }
     };
 
+    let bitcoin_fee_rate_per_vbyte = warp::any().map({
+        let bitcoin_fee_rate_per_vbyte = settings.bitcoin.fees.sat_per_vbyte;
+        move || bitcoin_fee_rate_per_vbyte
+    });
```

Comment:
In other routes, I just passed the complete settings previously. That results in less code churn if we wanted to access other things as well in those routes.